cookbook/how_to_upgrade_to_new_version.rst

```diff
@@ -0,0 +1,224 @@
+.. index::
+    single: Upgrade
+
+How to Upgrade to a New Version
+===============================
+
+1. Checkout from the GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve a new version and upgrade your OroCRM instance, please execute the following steps:
+
+**1** Create backups of your Database and Code.
+
+**2** "Cd" to the crm root folder and switch the application to the maintenance mode.
+
+.. code-block:: bash
+
+    $ cd /path/to/application
+    $ sudo -uwww-data app/console lexik:maintenance:lock --env prod
+
+**3** Stop the cron tasks
```

Comment:
Should be checked is it necessary to stop the cron in case when we put system to the maintenance mode with app/console lexik:maintenance:lock. If I remember correctly cron will not perform any actions while maintenance mode is turned on.

According to this code https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/CronBundle/Command/CronCommand.php#L45-L51 cron shouldn't run tasks in maintenance.